From 23807c4ff58ecd80d12badd5cbb94a054d194108 Mon Sep 17 00:00:00 2001
From: Muthukumar K <muthukumar.kandasamy@intel.com>
Date: Mon, 30 Jul 2012 11:07:52 -0400
Subject: [PORT FROM MAIN] Add new WIDI Device ID

BZ: 49306

WiDi OUT device ID clashes with the device ID of one of the
IN devices. This results in audio record failure. So
changing WiDi device ID to an unused value.

Change-Id: I7b56ddfdbc616295278dc5bbb6785c426144d054
Orig-Change-Id: I51087e4d66077cdeacb653b36e5f0c9a702da294
Category: feature differentiation
Domain: AUDIO_MEDIA-WIDI
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: Muthukumar K <muthukumar.kandasamy@intel.com>
Reviewed-on: http://android.intel.com:8080/79518
Reviewed-by: M, Arulselvan <arulselvan.m@intel.com>
Tested-by: M, Arulselvan <arulselvan.m@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/system/audio.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/system/audio.h b/include/system/audio.h
index c7cd109..3732b47 100644
--- a/include/system/audio.h
+++ b/include/system/audio.h
@@ -442,6 +442,7 @@ enum {
     /* FM transmitter out */
     AUDIO_DEVICE_OUT_FM                        = 0x100000,
     AUDIO_DEVICE_OUT_DEFAULT                   = AUDIO_DEVICE_BIT_DEFAULT,
+    AUDIO_DEVICE_OUT_WIDI                      = 0x1000000,
     AUDIO_DEVICE_OUT_ALL      = (AUDIO_DEVICE_OUT_EARPIECE |
                                  AUDIO_DEVICE_OUT_SPEAKER |
                                  AUDIO_DEVICE_OUT_WIRED_HEADSET |
-- 
1.7.4.1

